Tests/Marketplace/upload_packs.py

```diff
@@ -399,7 +399,6 @@ def print_packs_summary(packs_list):
         print_error(f"Number of failed packs: {len(failed_packs)}")
         failed_packs_table = _build_summary_table(failed_packs)
         print_error(failed_packs_table)
-        sys.exit(1)
```

Comment:
intentional?

Reply:
I re-added it from Ika's PR. - https://github.com/demisto/content/pull/6647

If we don't need it, I'll remove it. I more or less reverted everything.

Reply:
@amshamah419 leave `sys.exit(1)`. From now, we want to fail the build if there are failed in upload packs

Reply:
Makes sense. I'll re-add it